user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Security_Cassandra_TLS.md

```diff
@@ -77,8 +81,12 @@ To generate the certificates, you will need two tools: *openssl* and the *Java k
 1. Now that you have certificates for every node, digitally sign them with the private key of the root CA certificate. To do so, first create a certificate signing request (CSR).
 
    ```txt
-   keytool -certreq -keystore <NODE IP>.jks -alias <NODE IP> -file <NODE IP>.csr -keypass <STRONG PASSWORD> -storepass <STRONG PASSWORD> -ext SAN=DNS:<NODE NAME>,IP:<NODE IP>
+   keytool -certreq -keystore <HOSTNAME>.jks -alias <HOSTNAME> -file <HOSTNAME>.csr -keypass <STRONG PASSWORD> -storepass <STRONG PASSWORD> -ext SAN=DNS:<NODE NAME>,IP:<NODE IP>
    ```
+   
+> [!NOTE]
+> It's advised to use hostname instead of the IP. If you choose the IP over the hostname, you have to make sure the IP can be resolved, or the SAN has the correct hostname.
```

Comment:
Even with SAN, there should be some DNS resolution linking the IP to the CN of the cert (cassandra always does a hostname lookup)